src/services/cms/PrismicPrivacyTermsContent.js

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
Sorry can i just ask - why do we have a file here called `PrismicPrivacyTermsContent.js`?

Reply:
@IAmKio Earlier the logic of (fetching, mapping) was done in services/prismic-> queryDocumentsByID function but after interjection, made the function generic and moved all the fetching/mapping in the PrismicPrivacyTermsContent.js file and just use the output of function in PrismicDocumentModel to show the content.